bin/git/git.js

```diff
@@ -208,15 +213,16 @@ async function step(
 }
 
 export async function getBaselineCommits(client, { branch, ignoreLastBuildOnBranch = false } = {}) {
-  const { committedAt } = await getCommit();
+  const { commit, committedAt } = await getCommit();
 
   // Include the latest build from this branch as an ancestor of the current build
   const {
-    app: { firstBuild, lastBuild },
+    app: { firstBuild, lastBuild, pullRequest },
   } = await client.runQuery(TesterFirstCommittedAtQuery, {
     branch,
+    commit,
   });
-  debug(`App firstBuild: ${firstBuild}, lastBuild: ${lastBuild}`);
+  debug(`App firstBuild: ${firstBuild}, lastBuild: ${lastBuild}, pullRequest: ${pullRequest}`);
```

Comment:
Probably this is just going to yield `[object Object]`. Same goes for the first/last build of course.